crates/ghost_actor/src/ghost_channel.rs

```diff
@@ -452,11 +456,26 @@ impl<
                         request_id,
                         payload,
                     } => {
+                        let responder: GhostResponder<RequestToSelfResponse, Error> = match request_id {
+                            Some(request_id) => {
+                                let sender = self.sender.clone();
+                                Box::new(move |payload| {
+                                    sender.send(GhostEndpointMessage::Response {
+                                        request_id,
+                                        payload,
+                                    }).expect("should send");
+                                })
+                            }
+                            None => Box::new(|_|{}),
```

Comment:
This kind of worries me.  At the very least it should log an unexpected code path, maybe panic?